flows/flows.bzl

```diff
@@ -215,6 +215,13 @@ def _run_step_with_inputs(ctx, step, inputs_dict, outputs_step_dict):
 
     return dicts.add(inputs_dict, outputs_dict)
 
+def target_to_file(target):
+    all_files = target.files.to_list()
+    if len(all_files) != 1:
+        fail("Input target", target, "should provide exactly one file instead of", all_files)
```

Comment:
Is a space needed between the quoted values and the variables?... "Input target "?
Should you mention the length or did you want to list the files?